src/components/Execution/index.jsx

```diff
@@ -2,128 +2,86 @@ import React, { useCallback, useEffect, useState } from 'react'
 
 import Spinner from 'cozy-ui/react/Spinner'
 import Button from 'cozy-ui/react/Button'
-import { queryConnect, useClient } from 'cozy-client'
-import { sharesQuery } from 'doctypes'
+import { useClient } from 'cozy-client'
 import Webhook from './Webhook'
-import Share from './Share'
 
-export const Analyze = ({ shares }) => {
+export const Execution = () => {
   const client = useClient()
 
   const [isWorking, setIsWorking] = useState(false)
-  const [webhooks, setWebhooks] = useState()
-
-  const { data } = shares
-
-  const fetchWebhooks = useCallback(
-    async () => {
-      const webhooks = await client.stackClient.fetchJSON(
-        'GET',
-        '/jobs/triggers'
-      )
-
-      setWebhooks(
-        webhooks.data
-          .filter(hook => hook.attributes.type === '@webhook')
-          .sort((a, b) => a.id > b.id)
-      )
-    },
-    [client, setWebhooks]
-  )
-
-  useEffect(
-    () => {
-      fetchWebhooks()
-    },
-    [fetchWebhooks]
-  )
+  const [webhooks, setWebhooks] = useState([])
 
   const createWebhooks = useCallback(
     async () => {
-      const createBody = argument => ({
-        data: {
-          attributes: {
-            type: '@webhook',
-            worker: 'service',
-            arguments: argument
+      const query = async argument => {
```

Comment:
`argument` might be misleading now, as it does not correspond to the trigger argument. I would prefer something like `ǹame` or `triggerName`